package.json

```diff
@@ -108,7 +108,9 @@
     "path-to-regexp": "2.4.0",
     "popper.js": "1.14.3",
     "react-sortable-hoc": "^1.10.0",
+    "react-styled-frame": "^1.1.0",
     "react-transition-group": "2.5.3",
+    "styled-components": "5.0.0-beta.8",
```

Comment:
`beta.8`, scares me. I'm guessing we are going to bump this eventually?

Reply:
yes, the goal was to start on this PR with the latest. I expect that v5 will be out when we'll be ready to move forward with our PR. 

beta-9 was out 10 days ago and 4.4 is the last release before v5